elasticsearch-stress-test.py

```diff
@@ -316,60 +298,68 @@ def print_stats():
     print("")
 
 
-def print_stats_worker():
-
+def print_stats_worker(STARTED_TIMESTAMP):
     # Create a conditional lock to be used instead of sleep (prevent dead locks)
     lock = Condition()
 
     # Acquire it
     lock.acquire()
 
     # Print the stats every STATS_FREQUENCY seconds
-    while not has_timeout():
+    while (not has_timeout(STARTED_TIMESTAMP)) and (not shutdown_event.is_set()):
 
         # Wait for timeout
         lock.wait(STATS_FREQUENCY)
 
         # To avoid double printing
-        if not has_timeout():
-
+        if not has_timeout(STARTED_TIMESTAMP):
             # Print stats
-            print_stats()
+            print_stats(STARTED_TIMESTAMP)
 
 
 def main():
-    # Define the globals
-    global documents_templates
-    global indices
-    global STARTED_TIMESTAMP
-    global es
+    clients = []
+    all_indecies = []
 
-    try:
-        # Initiate the elasticsearch session
-        es = Elasticsearch(ES_ADDRESS)
+    # Set the timestamp
+    STARTED_TIMESTAMP = int(time.time())
 
-    except:
-        print("Could not connect to elasticsearch!")
-        sys.exit(1)
+    for esaddress in args.es_address:
+        print("")
+        print(esaddress)
```

Comment:
print("Starting initialization of {0}".format(esaddress)) or something?
Just a bit more info :)